core/Pos/Crypto/SafeSigning.hs

```diff
@@ -37,14 +39,15 @@ import           Universum
 import           Pos.Binary.Class      (Bi, Raw)
 import qualified Pos.Binary.Class      as Bi
 import           Pos.Crypto.Hashing    (Hash, hash)
+import qualified Pos.Crypto.Scrypt     as S
 import           Pos.Crypto.Signing    (ProxyCert (..), ProxySecretKey (..),
                                         PublicKey (..), SecretKey (..), Signature (..),
                                         sign, toPublic)
 import           Pos.Crypto.SignTag    (SignTag (SignProxySK), signTag)
 
 data EncryptedSecretKey = EncryptedSecretKey
     { eskPayload :: !CC.XPrv
```

Comment:
Though it's not mostly related to this change, can you put two documentation lines on these fields? I'm not sure i understand how this whole framework works :(
Also why field with type `EncryptedPass` is named `eskHash`? And why payload has type of a private key?